dump-objects

```diff
@@ -20,6 +20,8 @@ readonly min_expected_kinds=98
 
 default_output_dir=/data/k8s-backup
 
+DEBUG=false
```

Comment:
This cannot be overwritten from outside

Reply:
You are right. I rewrote this and added another parameter to the script. Afaik this makes more sense like this